eng/scripts/docs/tests/Service-Readme-Generation-Tests.ps1

```diff
@@ -0,0 +1,44 @@
+<#
```

Comment:
We should add a pipeline for to run these tests similar to what you did in the tools. I'd suggest putting the yml file directly in the tests folder and trigger it if anything under the docs folder changes.

Reply:
Create an issue to track the pre-requisite step of adding test pipelines. 
https://github.com/Azure/azure-sdk-tools/issues/4756